src/NBitcoin/Network.cs

```diff
@@ -324,32 +108,45 @@ public byte[] MagicBytes
             }
         }
 
+        public uint GenesisTime { get; protected set; }
+
+        public uint GenesisNonce { get; protected set; }
```

Comment:
This fields are probably not required to be part of network
GenesisNonce /GenesisBits/GenesisVersion /GenesisReward